web/modules/custom/asu_admin_toolbox/config/install/system.action.solr_reindex_children.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: solr_reindex_children
+label: 'Solr Reindex Children'
```

Comment:
i think this should be clearer that it reindexes the item itself and its children

Reply:
will update the label